README.md

```diff
@@ -36,6 +37,21 @@ $ make
 ### Using the ISO
 - The ISO can be used with [minishift](https://github.com/minishift/minishift) with the help of `--iso-url` i.e. `--iso-url=file:///$PATHTOISO`.
 
+<a name="tests"></a>
+## Tests
+
+Tests are written as Python scripts under `tests` directory and run through [Avocado](avocado-framework.readthedocs.io) framework. Ensure you have _Avocado_ installed by following [Avocado instllation guide](http://avocado-framework.readthedocs.io/en/44.0/GetStartedGuide.html#installing-avocado) before running any tests.
```

Comment:
I am confused by the paste:

> dhcp-0-106:~ cdkqe$ avocado -v
> Avocado 36.0lts

So at this stage _Avocado_ seems already installed

> dhcp-0-106:~ cdkqe$ pip install avocado-framework

and now avocado-framework is installed? What's that.

I don't make having OS X instructions a requirement to merge this, but at some stage I would appreciate if someone provides some re-producable instructions.